fastlane/lanes/release.rb

```diff
@@ -262,18 +332,40 @@ def trigger_buildkite_release_build(branch:, beta:)
   buildkite_annotate(style: 'info', context: 'trigger-release-build', message: message)
 end
 
-def create_release_backmerge_pr(version_to_merge:, next_version:)
+def create_backmerge_pr!
+  pr_urls = create_backmerge_prs
+
+  return pr_urls unless pr_urls > 1
+
+  backmerge_error_message = UI.user_error! <<~ERROR
+    Unexpectedly opened more than one backmerge pull request. URLs:
+    #{pr_urls.map { |url| "- #{url}" }.join("\n")}
+  ERROR
+  buildkite_annotate(style: 'error', context: 'error-creating-backmerge', message: backmerge_error_message) if is_ci
+  UI.user_error!(backmerge_error_message)
+end
+
+# Notice the plural in the name.
+# The action this method calls may create multiple backmerge PRs, depending on how many releas branches with version greater than the source are in the remote.
+def create_backmerge_prs
```

Comment:
Interesting approach. I wonder if this is really safe to assume and, if so, if this approach should be ported to other apps.

Reply:
@iangmaia [the `create_release_backmerge_pull_request` action from `release-toolkit` is already the one taking care of creating multiple PRs if there are multiple `release/*` branches with a greater version](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb#L61-L73). So no need to port anything to other repos, it's already handled as long as you use the toolkit action.

Reply:
Yeah I know, but I meant **throwing an error when there are multiple (unexpected) PRs**, which is what @mokagio is doing differently here.